.buildconfig-android.yml

```diff
@@ -1,4 +1,4 @@
-libraryVersion: 96.1.2
+libraryVersion: 96.1.3
```

Comment:
Hmm this is tricky but I like it - usually we only merge back the changelog since technically the latest release from `main` is still the 96.1.2... It's a tricky place to be in, and I'm not really sure what our "policy" is for this 😅 

But I do like this in this case, because there haven't been any other releases since 96.1.2 and this way the next release is 96.1.4 - so I saw we go for it!